src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -302,15 +305,32 @@ const CurationPage = (props: ICurationPageProps) => {
                 </Col>
               </Row>
             ))}
-        </>
+        </div>
+      )}
+      {props.vusData && (
+        <VusTable
+          vusObjList={props.vusData}
+          hugoSymbol={hugoSymbol}
+          account={props.account}
+          handleFirebaseUpdate={props.updateVus}
+          handleFirebaseDelete={props.deleteVus}
+          handleFirebaseCreateAnyValue={props.createAnyValueVus}
```

Comment:
I think you can move the store props down from CurationPage to the VusTable component. You would need to use `componentInject` on the VusTable component. Something like below:

```
const VusTable = (props) => {...}

const mapStoreToProps = ({ firebaseVusStore }: IRootStore) => ({
  vusData: firebaseVusStore.data,
  addVusListener: firebaseVusStore.addListener,
  ...
});

type StoreProps = Partial<ReturnType<typeof mapStoreToProps>>;

export default componentInject(mapStoreToProps)(observer(VusTable));
```

Reply:
We use `Partial` for the type because we want to make all the store props as optional because mobx will inject it for us. Without the optional, whenever you use `VusTable` it will complain that you need to pass in the props.